acceptance/localcluster/localcluster.go

```diff
@@ -137,7 +139,7 @@ func Create(numNodes int, stopper chan struct{}) *Cluster {
 		client:       newDockerClient(),
 		stopper:      stopper,
 		Nodes:        make([]*Container, numNodes),
-		UseTestCerts: true,
+		UseTestCerts: false,
```

Comment:
Yup. I'll remove this later though. I'll keep the focus of this PR on fixing the broken tests.